rc/lsp.kak

```diff
@@ -120,8 +118,8 @@ method   = "textDocument/didChange"
 draft    = """
 %s"""
 ' "${kak_session}" "${kak_buffile}" "${kak_opt_filetype}" "${kak_timestamp}" "${lsp_draft}" | eval ${kak_opt_lsp_cmd} --request
-printf %s "${kak_opt_lsp_callback}" | kak -p "${kak_session}"
-) > /dev/null 2>&1 < /dev/null & }
+printf %s "${kak_opt_lsp_callback}"
```

Comment:
Wow, I just realized why this broke rls completions - it's because we stop sending the callback.

The current lsp-did-change-then did:

    evaluate-commands -save-regs '|' %{
        set-register '|' %{
            echo ... | kak-lsp --request >/dev/null 2>&1 </dev/null &
            printf %s\\n "${kak_opt_lsp_callback}" | kak -p "${kak_session}"
        }
        execute-keys -draft '%<a-|><ret>'
    }

your version does

    evaluate-commands -save-regs '|' %{
        set-register '|' %{
            echo ... | kak-lsp --request
            printf %s\\n "${kak_opt_lsp_callback}"
        }
        execute-keys -draft '%<a-|><ret>'
    }

`<a-|>` throws away its output, so kak_opt_lsp_callback is not run.

The correct solution is much simpler, I'll push it in a bit.
I'll test a bit more and merge seen.

(I found this because I needed a synchronouse lsp-did-change-and-then for https://github.com/kak-lsp/kak-lsp/issues/582)